### ISSUE TITLE
“Test your skills: Advanced styling” doc: "Try updating the live code below to recreate the finished example:" but there are no examples to be found

### ISSUE THREAD

**Issue by yomizana** [mdn/content#15834]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Advanced_styling

### What specific section or headline is this issue about?

Advanced form styling 1, 2, and 3

### What information was incorrect, unhelpful, or incomplete?

All exercises say "Try updating the live code below to recreate the finished example:" but there are no examples to be found. 

### What did you expect to see?

The examples we are supposed to recreate.

### Do you have any supporting links, references, or citations?

The same thing occurs with exercises in other pages:
[Test your skills: Basic controls](https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Basic_controls)
[Test your skills: HTML5 controls](https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_HTML5_controls)
[Test your skills: Other controls](https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Other_controls)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/test_your_skills_colon__advanced_styling`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Advanced_styling
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/test_your_skills_colon__advanced_styling/index.md
* Last commit: https://github.com/mdn/content/commit/2777cdc8c8b54268a24beeeb7a98520e3331d3be
* Document last modified: 2022-04-26T11:32:33.000Z

</details>

**Comment by vagadeeshwar** [comment]
Hey, I think by finished example they ask you to follow the mentioned steps and build the basic styled element they have given you in the livecode.
And not an image of the finished styling.

**Comment by yomizana** [comment]
That would make no sense though, the live code contains the starting point of the task and the point is to modify it to recreate the supposed example. If it is as you say, then the whole exercise needs to be reworded to make it clear how things are supposed to look at the end, otherwise I wouldn't know if I'm doing a good job or not.

Check out these two pages, which do have the examples we're supposed to recreate:
[Test your skills: sizing](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Sizing_tasks)
[Test your skills: Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox_skills)

**Comment by Leo-U** [comment]
I agree with @yomizana. "finished examples" means there are supposed to be finished examples for us to look at. This issue affects multiple pages such as [here](https://developer.mozilla.org/en-US/docs/Learn/Forms/Test_your_skills:_Basic_controls), [here](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Test_your_skills:_Functions,), and [here](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Test_your_skills:_Conditionals), and presumably others.

Here are screenshots of discord comments from other people encountering the same issue:
[1](https://github.com/mdn/content/assets/47509535/214ae53a-2b26-4423-ba8e-9ec0c9879a38)
[2](https://github.com/mdn/content/assets/47509535/23b72550-634e-44a0-b1cf-0ceb2e12d97d)

**Comment by bsmth** [comment]
Chris, you're working on these at the moment, just tagging you to check if we can close this when we've finished.
